e2e/cypress/integration/channel/mobile_profile_popover_spec.js

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+describe('Profile popover', () => {
+    before(() => {
+        // # Go to the top public channel with "user-1"
+        cy.apiLogin('user-1');
+        cy.toTopPublicChannelView();
+        cy.viewport('iphone-6');
+    });
+
+    it('M18715 Profile popover should render', () => {
+        cy.getLastPostId().then((postId) => {
```

Comment:
This code assumes that the top channel has posts in it. Generally, it should, but can you send a post to guarantee that there will be a post for you to look up?

Reply:
Ah yep, definitely can do